src/katello/packages.py

```diff
@@ -18,9 +18,10 @@ def upload_package_profile(force=False):
 
 
 def purge_package_cache():
-    try:
-        os.remove(PACKAGE_CACHE_FILE)
-    except OSError:
-        error = sys.exc_info()[1]  # backward and forward compatible way to get the exception
-        if error.errno is not errno.ENOENT:
-            sys.stderr.write("Unable to clear the package cache")
+    for cache_file in [PACKAGE_CACHE_FILE, COMBINED_PROFILE_CACHE_FILE]:
+        try:
+            os.remove(cache_file)
+        except OSError:
+            error = sys.exc_info()[1]  # backward and forward compatible way to get the exception
+            if error.errno is not errno.ENOENT:
+                sys.stderr.write("Unable to clear the package cache: " + cache_file)
```

Comment:
Since we were only looking for one file (and assuming it would be there) I guess it *was* an error if it wasn't deleted. But now that there are two possibilities if one is missing it's probably not an error. Maybe the try block should check for existence of the file before attempting delete so we'd raise the error on the condition where it existed but couldn't be deleted?

Also looks like the unit test broke 😢